modules/administration-guide/pages/monitoring-che.adoc

```diff
@@ -1,24 +1,15 @@
 :_content-type: ASSEMBLY
-:navtitle: Monitoring {prod-short}
+:navtitle: Monitoring {prod-short} server
```

Comment:
```suggestion
:navtitle: Monitoring the {prod-short} server
```

Reply:
Since [Che Server](https://github.com/eclipse-che/che-server) is a proper noun, I went with `Monitoring {prod-short}` Server, how does that sound?

Reply:
I'll ask another technical writer for his take on this and then let you know. (I don't have a preference.)